versions.cfg

```diff
@@ -263,21 +262,21 @@ requests-toolbelt = 1.0.0
 responses = 0.25.7
 rfc3986 = 2.0.0
 rich = 14.0.0
-robotframework = 6.0.2
-# robotframework >= 6.1 is only supported with robotframwork-lsp >= 1.11.0,
-# but https://github.com/robocorp/robotframework-lsp/issues/947
+robotframework = 7.3.2
 robotframework-assertion-engine = 3.0.3
-robotframework-browser = 19.3.1
+robotframework-browser = 19.8.0
 robotframework-debuglibrary = 2.5.0
-robotframework-lsp = 1.10.1
+robotframework-lsp = 1.13.0
 robotframework-pythonlibcore = 4.4.1
+# core tests do not use selenium anymore -> remove?
 robotframework-selenium2library = 3.0.0
 robotframework-selenium2screenshots = 0.8.1
-robotframework-seleniumlibrary = 6.1.3
+robotframework-seleniumlibrary = 6.7.1
 robotframework-seleniumtestability = 2.1.0
+robotsuite = 3.0.0rc1
```

Comment:
I prefer no release candidates in 6.1, but for a test dependency it should be fine.  I will ask for a release.